.github/workflows/doc.yml

```
@@ -53,6 +53,27 @@ jobs:
     - name: 'Build HTML documentation'
       run: make -C Doc/ SPHINXOPTS="-q" SPHINXERRORHANDLING="-W --keep-going" html
 
+    # Add pull request annotations for Sphinx nitpicks (missing references)
+    - name: Get list of changed files
+      id: changed_files
+      uses: Ana06/get-changed-files@v2.2.0
+    - name: 'Build changed files with nitpicks'
+      continue-on-error: true
+      run: |
+        # Mark files the pull request modified
+        touch ${{ steps.changed_files.outputs.added_modified }}
+        # Build docs with the '-n' (nit-picky) option, convert warnings
+        make -C Doc/ PYTHON=../python SPHINXOPTS="-q -n --keep-going" html 2>&1 |
+            python Doc/tools/warnings-to-gh-actions.py
+
+    # Ensure some files always pass Sphinx nitpicks (missing references)
```

```suggestion
    # Ensure some files always pass Sphinx nitpicky mode (no missing references)
```

Use more precise and clearer terminology here